Makefile

```diff
@@ -10,22 +10,38 @@ test:
 	@echo "===================================================================="
 	@echo "Testing Connection Type TCP with all features"
 	@echo "===================================================================="
-	@REDISRS_SERVER_TYPE=tcp cargo test --all-features -- --nocapture --test-threads=1
+	@REDISRS_SERVER_TYPE=tcp cargo test --features test_default -- --nocapture --test-threads=1 --skip test_json
 
 	@echo "===================================================================="
 	@echo "Testing Connection Type TCP with all features and TLS support"
 	@echo "===================================================================="
-	@REDISRS_SERVER_TYPE=tcp+tls cargo test --all-features -- --nocapture --test-threads=1
+	@REDISRS_SERVER_TYPE=tcp+tls cargo test --features test_default -- --nocapture --test-threads=1 --skip test_json
 
 	@echo "===================================================================="
 	@echo "Testing Connection Type UNIX"
 	@echo "===================================================================="
-	@REDISRS_SERVER_TYPE=unix cargo test --test parser --test test_basic --test test_types --all-features -- --test-threads=1
+	@REDISRS_SERVER_TYPE=unix cargo test --test parser --test test_basic --test test_types --features test_default -- --test-threads=1
 
 	@echo "===================================================================="
 	@echo "Testing Connection Type UNIX SOCKETS"
 	@echo "===================================================================="
-	@REDISRS_SERVER_TYPE=unix cargo test --all-features -- --skip test_cluster
+	@REDISRS_SERVER_TYPE=unix cargo test --features test_default -- --skip test_cluster --skip test_json
+
+test-json:
```

Comment:
by separating the tests for this feature into a different command set entirely we allow future additions (perhaps redisearch for example) and this makefile would be ready to accommodate for them.